src/validation/custom.validation.ts

```diff
@@ -14,6 +16,13 @@ export const checkMaxFieldIfRadioButtonSelected = (selected: boolean, errMsg: st
   return true;
 };
 
+export const checkInvalidCharactersIfRadioButtonSelected = (selected: boolean, errMsg: string, value: string) => {
+  if (value && !(selected || value.match(VALID_CHARACTERS))) {
```

Comment:
You have to check it the value is truthy if you done it will throw an error we don't want this we only want to throw it when something invalid has been entered.

Reply:
Not sure it it were falsey then its length would be < max value so it is covered by that test.

Reply:
I think the logic is the wrong way around. Looking at checkMaxFieldIfRadioButtonSelected, it should throw the error message if the string is too long. 
For checkInvalidCharactersIfRadioButtonSelected - 
checkInvalidCharactersIfRadioButtonSelected(true, "err", "Путин") - returns true (as in it the text is ok)
checkInvalidCharactersIfRadioButtonSelected(false, "err", "Путин") - throws error - (selected is false so we shouldn't be checking the field)